core/src/test/java/org/apache/brooklyn/core/mgmt/internal/EntityExecutionManagerTest.java

```diff
@@ -68,6 +68,7 @@
 import com.google.common.util.concurrent.Callables;
 
 /** Includes many tests for {@link BrooklynGarbageCollector} */
+@Test
```

Comment:
I'll take out the plain `@Test`s  - I take it, though, that it's better to leave the `@Test(groups="Integration")` as-is?